src/components/motor/MotorController.h

```diff
@@ -12,15 +12,32 @@ namespace Pinetime {
       MotorController() = default;
 
       void Init();
-      void RunForDuration(uint8_t motorDuration);
+      void RunForDuration(uint16_t motorDuration);
       void StartRinging();
       void StopRinging();
+      void StartWakeAlarm();
```

Comment:
I think the MotorController interface should be more generic and not specific for one app. Maybe a function which takes some kind of description of the on/off/strength/PWM pattern and controls the motor depending on that? Then an adapter could be used to implement the alarm domain functions.